tests/core/test_circuit_breaker.py

```
@@ -0,0 +1,199 @@
+        result = await circuit_breaker.execute(success_func)
+        assert result == "success"
+        assert circuit_breaker.stats.success_count == 1
+        assert circuit_breaker.stats.failure_count == 0
+
+    @pytest.mark.asyncio
+    async def test_failed_execution(self, circuit_breaker):
+        """Test failed function execution."""
+        async def fail_func():
+            raise ValueError("test error")
+
+        with pytest.raises(ValueError):
```

Ruff (0.8.2)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)